exercises/Toluwalase_Agoro/ex01_basics/src/main.cpp

```diff
@@ -0,0 +1,184 @@
+
+#include <iostream>
+#include <vector>
+#include <fstream>
+#include <string>
+#include <iomanip>
+using namespace std;
+
+
+void toLower(string &stringName) {
+	/* Function accepts a string and modifies string
+	by making it lowercase and returns nothing
+	*/
+	string PlaceHolderString{  };
```

Comment:
Please can you use camelCase for variable names ie placeHolderString. I'll just highlight this one but please can you change other variable names in this unit as well